homeassistant/components/starlink/sensor.py

```diff
@@ -122,6 +123,22 @@ def native_value(self) -> StateType | datetime:
         native_unit_of_measurement=PERCENTAGE,
         value_fn=lambda data: data.status["pop_ping_drop_rate"] * 100,
     ),
+    StarlinkSensorEntityDescription(
+        key="upload",
+        translation_key="upload",
+        device_class=SensorDeviceClass.DATA_SIZE,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        native_unit_of_measurement=UnitOfInformation.BYTES,
```

Comment:
Let's add a suggested unit of measurement as well.
For an every increasing sensor, this unit might be too small :)

Reply:
Sure thing!